class.application-passwords.php

```diff
@@ -119,8 +123,8 @@ public static function catch_submission( $user_id ) {
 
 	public static function catch_delete_application_password() {
 		$user_id = get_current_user_id();
-		if ( ! empty( $_REQUEST['delete_application_password'] ) ) {
-			$slug = $_REQUEST['delete_application_password'];
+		if ( ! empty( $_GET['delete_application_password'] ) ) {
```

Comment:
Depending on how the AJAX processing endpoints are written, this may be better left as $_REQUEST to let one endpoint do both.  Maybe.  I'd need to see it built out first.